src/legacy/dataSelectAction.ts

```diff
@@ -86,7 +86,7 @@ function handleSeriesLegacySelectEvents(
                         type: legacyEventName,
                         seriesId: seriesModel.id,
                         name: isArray(dataIndex) ? data.getName(dataIndex[0]) : data.getName(dataIndex),
-                        selected: extend({}, seriesModel.option.selectedMap)
+                        selected: extend({}, seriesModel.option as Dictionary<boolean>)
```

Comment:
Must be very careful when using `as`. Here the code is obviously wrong

Reply:
Sorry, my bad